docs/adguard-for-android/solving-problems/logcat.md

```diff
@@ -88,8 +94,4 @@ Send this log to us by contacting support or by any other way.
 
 5. Send this log to our support team.
 
-:::note
-
-Our support team will process your ticket much faster if you specify the HelpDesk ticket number or the GitHub issue number in your message to support.
-
-:::
+> Note: Our support team will process your ticket much faster if you specify the HelpDesk ticket number or the GitHub issue number in your message to support.
```

Comment:
oh no, please leave the admonitions (you can check documentation here: https://docusaurus.io/docs/markdown-features/admonitions), we no longer use quote marks

Reply:
>>> а это потому что текст из Настиного пра, видимо в апреле еще не было возможности юзать адмонишны? [2]